README.md

```diff
@@ -133,6 +133,17 @@ Add code to connect to the server and the database **myProject**:
 > However, all async API calls support an optional callback as the final argument,
 > if a callback is provided a Promise will not be returned.
 
+> **NOTE:** Resolving DNS Connection issues
+>
+> Node18 changed the default dns resolution ordering from ipv4 first always to the ordering
+> that dns provider returns them in.  In some environments, this can result in `localhost` resolving to
+> an ipv6 address instead of ipv4 and can result in a failure to connect to the server.
```

Comment:
```suggestion
> Node.js 18 changed the default DNS resolution ordering from always prioritizing ipv4 to the ordering
> returned by the DNS provider.  In some environments, this can result in `localhost` resolving to
> an ipv6 address instead of ipv4 and a consequent failure to connect to the server.
```